development/learning/NextJS-Server-Side-Rendering.md

```diff
@@ -0,0 +1,41 @@
+# ReactJS to NextJS Migration Using App Router In Server Side Rendering
```

Comment:
Please rewrite entire document with examples for anything that is being mentioned.